lib/js/styles/ItemsTypography.vue

```diff
@@ -1,25 +1,25 @@
 <template>
-	<div>
-		<div v-for="item in items" :key="item.id" class="itemsListItem">
-			<div v-if="item.label" class="itemsListItem__label">{{ item.label }}</div>
-			<div v-if="item.value" class="itemsListItem__value">{{ item.value }}</div>
-			<div v-if="item.ratio" class="itemsListItem__value">{{ item.ratio }}</div>
-			<div v-if="item.token" class="itemsListItem__token">{{ item.token }}</div>
-			<div v-if="item.attributesRaw" class="itemsListItem__attributes">
+	<table class="itemsListItem">
+		<tr v-for="item in items" :key="item.id" class="itemsListItem__row">
+			<td v-if="item.label" class="itemsListItem__label">{{ item.label }}</td>
+			<td v-if="item.value" class="itemsListItem__value">{{ item.value }}</td>
+			<td v-if="item.ratio || item.value" class="itemsListItem__value">{{ item.ratio }}</td>
```

Comment:
do we still need to render empty `<td>`?

Reply:
I removed it just now.